.github/workflows/build-test.yml

```diff
@@ -503,7 +503,7 @@ jobs:
 
       - name: Install screen capture dependencies
         if: matrix.os == 'ubuntu-latest'
-        run: sudo apt-get install imagemagick
+        run: sudo apt-get install imagemagick x11-xserver-utils
```

Comment:
This is what was missing and the reason why screenshots stopped working this week.
Looks like something changed somehwere upstream in the ubuntu images.